src/bin.ts

```diff
@@ -4,10 +4,15 @@ import {isBoolean} from './util';
 export interface BinBase {
   /**
    * The number base to use for automatic bin determination (default is base 10).
+   *
+   * __Default value:__ `10`
+   *
    */
   base?: number;
   /**
    * An exact step size to use between bins. If provided, options such as maxbins will be ignored.
+   *
+   * __Note:__ If provided, options such as maxbins will be ignored.
```

Comment:
This description now contains "If provided, options such as maxbins will be ignored." twice.

Either remove this line or remove it from the above line?